src/Features/Inventoriable.php

```diff
@@ -73,7 +73,12 @@ public function getInventoryFileName(bool $prepend_dir_path = true): ?string
         $source = new \AutoUpdateSystem();
         $source->getFromDBByCrit(['name' => 'GLPI Native Inventory']);
```

Comment:
Unless I am wrong, fetching `$source` is not usefull if there is no `autoupdatesystems_id` field on item. So source fetching should be skipped if variable is not used.